apps/site/src/lib/components/forms/Slider.tsx

```diff
@@ -0,0 +1,63 @@
+import RequiredAsterisk from "./RequiredAsterisk";
+
+interface SliderProps {
+	pretext?: string;
+	postText?: string;
+	name: string;
+	containerClass: string;
+	labelText: string;
+}
+
+interface SliderTickProps {
+	leftPercentage: string;
+	label: string;
+}
+
+const SliderTick = ({ leftPercentage, label }: SliderTickProps) => (
+	<div className="absolute h-full" style={{ left: leftPercentage }}>
+		<div className="rounded bg-white h-full w-[4px]" />
+		<p className="absolute mt-4 left-[-50%]">{label}</p>
+	</div>
+);
+
+export default function Slider({
+	name,
+	pretext,
+	postText,
+	containerClass,
+	labelText,
+}: SliderProps) {
+	return (
+		<div className={containerClass}>
+			<p className="m-0 text-lg mb-4">
+				{labelText} <RequiredAsterisk />
+			</p>
+			<div className="flex gap-4 w-full">
+				<p className="hidden md:block text-center m-0">{pretext}</p>
+				<div className="relative flex items-center min-h-[50px] w-full">
+					<SliderTick leftPercentage="0.7%" label="1" />
+					<SliderTick leftPercentage="25.2%" label="2" />
+					<SliderTick leftPercentage="49.7%" label="3" />
+					<SliderTick leftPercentage="74%" label="4" />
+					<SliderTick leftPercentage="98.7%" label="5" />
```

Comment:
Question: is there a particular reason these are not aligned to every 25%?

Reply:
When testing, the ticks did not line up with the slider thumb at exactly 25% intervals, at least on Chrome. 

I felt like it should have lined up as the slider is set to full width and the slider ticks are also in that full width div.

I also have no way to test how the thumb behaves on Webkit, so it might be different there too.